docs/javascript/v2/how--to-guides/debugging/faq.mdx

```diff
@@ -168,3 +168,37 @@ You can get the HLS URL in several ways based on whether you use 100ms client SD
 #### I get type errors related to WebRTC Stats, for example, 'Cannot find name RTCInboundRtpStreamStats'.
 
 The minimum supported version of TypeScript for the SDK is '4.4'. You could update to any version above 4.4 or if you are on an old version of TypeScript and cannot do a major upgrade you can set `skipLibCheck: true` in your tsconfig file.
+
+#### Why is my screenshare not working on mobile browsers?
+
+The screenshare API(getDisplayMedia) is not yet supported on mobile browsers. Refer [this](https://caniuse.com/?search=getDisplayMedia) for supported browsers.
+
+
+#### Can I have multiple instances of HMSRoomProvider each mapping to a different room?
```

Comment:
i think the question is more of an answer.
suggestion - 

Question - "Can I use the sdk to join multiple rooms at the same time?"

Answer - "Yes. Although one instance of SDK can be used to join only one room, you can create multiple instances of the SDK which can operate independently. 
If you're using react-sdk, you can use multiple instances of HMSRoomProvider. A component using hooks from react-sdk will reach out to the nearest HMSRoomProvider in their ancestor tree. For e.g.
```
const App = () => {

    return (
        <Fragment>
          <HMSRoomProvider>
            <Conference room="room1" />
          </HMSRoomProvider>
          <HMSRoomProvider>
            <Conference room="room2" />
          </HMSRoomProvider>
        </Fragment>
    )
}
```